ui/widgets/nuxeo-user-avatar.js

```diff
@@ -50,12 +52,23 @@ import { html } from '@polymer/polymer/lib/utils/html-tag.js';
             transform: translate(-50%, -50%);
             text-transform: uppercase;
           }
+
+          iron-icon {
+            position: absolute;
+            top: 50%;
+            left: 50%;
+            transform: translate(-50%, -50%);
+            height: 12px;
+            width: 12px;
+            fill: white;
+          }
         </style>
 
         <nuxeo-resource id="getUserProfile" enrichers="userprofile" enrichers-entity="user"></nuxeo-resource>
 
         <div id="container">
-          <span id="character">{{_output}}</span>
+          <span id="character" hidden$="[[!_isInTheAlphabet]]">{{_output}}</span>
```

Comment:
Not want to be picky but the name "isInTheAlphabet" is not completely "self-explanatory". What is, or not, in the alphabet? May I suggest something like "containsSpecialChars" or "nameContainsNonLatinChars". WDYT?

Reply:
I choose "isInTheAlphabet" because there is already a const called "alphabet" that holds all the values it expects. I think this might be a natural choice in the context of the current code. I don't agree with the "containsSpecialChars" because we shouldn't consider some languages to be more special than the others. Since we are in fact checking if the value is in the const "alphabet", for me it makes sense this choice.